Logger/LogFactory.cs

```diff
@@ -2,10 +2,27 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string FilePath { get; set; } = "";
+        public BaseLogger? CreateLogger(string className)
         {
-            
-            return null;
+            if(FilePath == "")
+            {
+                return null;
+            }
+
+            BaseLogger fileLog = new FileLogger()
+            {
+                ClassName = className,
+                FilePath = this.FilePath
```

Comment:
```suggestion
                FilePath = _FilePath
```
Avoid using `.this`. Also, you may consider passing your file path through the constructor rather than as an auto property.